src/clr-angular/forms/tests/container.spec.ts

```diff
@@ -53,13 +53,16 @@ export function ReactiveSpec(testContainer, testControl, testComponent, wrapperC
   fullSpec('reactive', testContainer, testControl, testComponent, wrapperClass);
 }
 
-function fullSpec(description, testContainer, testControl, testComponent, wrapperClass) {
+function fullSpec(description, testContainer, directives: any | any[], testComponent, wrapperClass) {
   describe(description, () => {
     let fixture, containerDE, container, containerEl, ifErrorService, layoutService;
+    if (!Array.isArray(directives)) {
+      directives = [directives];
+    }
     beforeEach(() => {
       TestBed.configureTestingModule({
         imports: [ClrIconModule, ClrCommonFormsModule, FormsModule, ReactiveFormsModule],
-        declarations: [testContainer, testControl, testComponent],
+        declarations: [testContainer, ...directives, testComponent],
```

Comment:
You're doing a spread on something of type `any | any[]`. I have no idea what the expected result is here. If you get something of type `any[]`, it's going to spread as expected, but if you get something of type `any`, don't you risk is spreading the actual properties of the object you receive? I'm guessing it works right now since you submitted the PR, I'm just really curious about it.

Reply:
Line 59 does a check if `directives` is an array, if not converts it to always be an array. This was a backward compatible API change I made since in the case of radio and checkboxes they will need multiple directives (wrappers and containers) and didn't want to hack all the previous uses. Happy to discuss other approaches, but Shijir's suggestion is a more clever one liner for what I did in lines 59-61.